libs/template/src/lib/core/render-aware/render-aware_creator.ts

```diff
@@ -73,30 +77,30 @@ export function createRenderAware<U>(cfg: {
       return observable$;
     }),
     // forward only observable values
-    filter(o$ => o$ !== undefined),
-    switchMap(o$ =>
-      o$.pipe(
-        distinctUntilChanged(),
-        tap(cfg.updateObserver),
-        currentStrategy.rxScheduleCD,
-        tap({
-          // handle "error" and "complete" notifications for Observable from template
-          error: err => {
-            console.error(err);
-            if (cfg.updateObserver.error) {
-              cfg.updateObserver.error(err);
-              currentStrategy.detectChanges();
-            }
-          },
-          complete: cfg.updateObserver.complete
-            ? () => currentStrategy.detectChanges()
-            : undefined
-        })
-      )
+    filter((o$) => o$ !== undefined),
+    distinctUntilChanged(),
+    switchMap((o$) =>
+      o$
+        // Added behavior will get applied to the observable in `renderWithLatestStrategy`
+        .pipe(
+          // Forward only distinct values
+          distinctUntilChanged(),
+          // Update completion, error and next
+          tap(cfg.updateObserver),
+          renderWithLatestStrategy(strategy$)
+        )
     ),
-    catchError(e => {
-      return EMPTY;
-    })
+    tap({
+      // TODO: doesnt work
```

Comment:
Maybe this will help: https://github.com/BioPhoton/rx-angular/pull/199

Reply:
https://github.com/BioPhoton/rx-angular/pull/199#issuecomment-670229987